packages/components/socialFeed/Map/Map.types.ts

```diff
@@ -0,0 +1,10 @@
+import { StyleProp, ViewStyle } from "react-native";
+
+import { CustomLatLngExpression, PostCategory } from "@/utils/types/feed";
+
+export interface MapProps {
+  creatingPostLocation?: CustomLatLngExpression; // When the user is adding a location to a post he's creating
+  creatingPostCategory?: PostCategory; // When the user is adding a location to a post he's creating
```

Comment:
twice same comment, should not be when the user is a adding a category to a post he's creating ?